bcbio/structural/__init__.py

```diff
@@ -130,6 +132,29 @@ def validate_sv(data):
     """
     return [[validate.evaluate(data)]]
 
+def create_cnv_pon(samples):
+    """create normal db using SNV PON and coverage information"""
+    coverage_files = []
+    sample0 = utils.to_single_data(samples[0])
+    purecn_pon_build = tz.get_in(["config", "algorithm", "purecn_pon_build"], sample0)
+    if purecn_pon_build:
+        for sample in (utils.to_single_data(x) for x in samples):
+            sample_coverage = tz.get_in(["depth", "bins", "purecn"], sample)
+            if os.path.exists(sample_coverage):
+                coverage_files.append(sample_coverage)
+        work_dir = tz.get_in(["dirs", "work"], sample0)
+        out_dir = utils.safe_makedir(os.path.join(work_dir, "gemini"))
+        purecn_coverage_files_txt = os.path.join(out_dir, "purecn_coverage_files.txt")
+        with open(purecn_coverage_files_txt, "w") as out_handle:
+            for fname in coverage_files:
+                out_handle.write(fname + "\n")
+        snv_pon = tz.get_in(["config", "algorithm", "purecn_snv_pon"], sample0)
+        normal_db = os.path.join(out_dir, "normalDB_hg38.rds")
+        # delete normal db in work/gemini if you'd like to rebuild it!
```

Comment:
This needs to be documented somewhere (if it has not been done already).

Reply:
A good point. bcbio's persistence, in general, is common knowledge among users, but I don't think it was documented. I have clarified it here: https://bcbio-nextgen.readthedocs.io/en/latest/contents/configuration.html#persistence